src/types/memory-controller.ts

```diff
@@ -1,12 +1,25 @@
+import axios from "axios";
 import { CardData, GameData } from "./data-models";
+import { GameDataDTO, GameResultDTO } from "./dtos";
 import { testData } from "./test-data";
 
+const baseURL = "/minigames/memory/api/v1";
+const configurationId = window.location.pathname.split("/").pop();
+
 export class MemoryController {
-  gameData: GameData;
+  postGameResult() {
+    const result: GameResultDTO = new GameResultDTO(
+      configurationId!,
+      window.localStorage.getItem("userId")!
+    );
+    console.log(result);
+    axios.post(`${baseURL}/results`, result).catch(function (error) {
+      console.log(error);
```

Comment:
In case the gameresult could not be saved, the user should be notified, not only console log.